template/makeBp.js

```diff
@@ -106,7 +106,7 @@ function getPropertyCppType(property, datatype) {
             } else if (isCollection) {
                 return "UPlayFabJsonObject* " + propSafeName + " = nullptr;";
             } else if (property.isenum) {
-                return "E" + getDataTypeSafeName(property, "actualtype") + " " + propSafeName + ";";
+                return "E" + getDataTypeSafeName(property, "actualtype") + " " + propSafeName + " = static_cast<E"+getDataTypeSafeName(property, "actualtype") + ">(0);";
```

Comment:
Also, add the spaces around the first added +

static_cast<E" + getDataTypeSafeName(property,

instead of 

static_cast<E"+getDataTypeSafeName(property,